boards/nxp/frdm_mcxn947/frdm_mcxn947_mcxn947_cpu0.dts

```diff
@@ -116,3 +120,9 @@
 &enet_mdio {
 	status = "okay";
 };
+
+&flexpwm1_pwm0 {
+	status = "okay";
+	pinctrl-0 = <&pinmux_flexpwm1_pwm0>;
+	pinctrl-names = "default";
+};
```

Comment:
Similar to other peripherals, have the pinctrl defines in the common dtsi file and enable the peripheral in the cpu0 file.

Reply:
I understand now, please check again to see if this is correct.

Reply:
Please fix a typo in the commit message. Change `inlcuding ` to `including`